interestingness/timedRun.py

```diff
@@ -10,7 +13,7 @@
 path0 = os.path.dirname(os.path.abspath(__file__))
 path1 = os.path.abspath(os.path.join(path0, os.pardir, 'interestingness'))
 sys.path.append(path1)
-import envVars  # noqa
+import envVars  # noqa, pylint: disable=relative-import,wrong-import-position
```

Comment:
Ditto re: `absolute_import`. In this case the sys.path stuff above probably isn't needed if you do.